chef_master/source/data_collection_habitat.rst

```diff
@@ -0,0 +1,49 @@
+Data Collection with Habitat
+=====================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/data_collection_habitat.rst>`__
+
+
+The data collection capabilities of Chef Automate can also be used to collect and report on Habitat ring data. The Prism Habitat package collects this data and sends it to a Chef Automate server's REST API endpoint. You can configure settings like the data collector URL, token, the Habitat supervisor used to get the ring information, and so on. For more information on the Prism package, see :doc:`Habitat Prism </habitat_prism>`. For more information on Habitat, see the `Habitat site <https://habitat.sh/>`__.
+
+
+Use an external Elasticsearch cluster (optional)
```

Comment:
I'm confused as to why this is relevant to a guide on habitat data collection - the relationship between this and the capabilities mentioned in the first paragraph isn't clear. This seems like it makes more sense as a subset of the data collection guide, with a link here - I don't think a topic as general as external ES should only exist in a habitat guide.

Reply:
It is legacy information.

Reply:
How so? This is still possible with ES5 and Automate 1.6..and even if it is legacy information (that some people are using) I don't think anyone would think to look for it in a guide that has Habitat in the title.